Taskfile.yml

```diff
@@ -3,21 +3,33 @@
 version: '3'
 
 env:
+  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2023"
   SRC_DIR: "{{.TASKFILE_DIR}}/src"
 
 tasks:
+  ci:
+    desc: Workflow to run in CI
+    cmds:
+      - task: workspace
+      - task: has-latest-opslevel-go
+      - task: install-gofumpt
+      - task: install-golangci-lint
+      - task: lint
+      - task: test
+
   lint:
```

Comment:
I noticed that this task fails trying to format and lint things in my submodules. Is that expected? 

```
~/r/kubectl-opslevel/src db/update-Ta…t-lint-fixes ❯ task lint                                Ruby 3.0.0 05:24:22 pm
task: [lint] test -z "$(gofumpt -d -e . | tee /dev/stderr)"
diff submodules/opslevel-common/signal.go.orig submodules/opslevel-common/signal.go
--- submodules/opslevel-common/signal.go.orig
+++ submodules/opslevel-common/signal.go
@@ -1,23 +1,26 @@
 package opslevel_common

 import (
-	"github.com/rs/zerolog/log"
 	"os"
 	"os/signal"
 	"syscall"
-)
```

Reply:
These have been fixed in `opslevel-common` but I'm adding `--skip-dirs submodules` to the lint call

Reply:
Actually this is the formatter, and it doesn't look like there's a non-convoluted way to skip these with that. But also this shouldn't be a problem anymore since the formatter is part of `opslevel-common`'s CI check